docs/advanced-usage/connecting-cloud-storage/s3-compatible.md

```diff
@@ -0,0 +1,78 @@
+---
+subtitle: MinIO, Oracle, Hitachi
+---
+
+Integrations can be established to S3-compatible systems (for example, MinIO).
+To get started, ensure you have admin access within Kolena.
+Navigate to your [Organization Settings's Integration Tab](https://app.kolena.io/redirect/organization?tab=integrations) and click "Add Integration", then "MinIO".
+
+Steps performed outside of Kolena are shown for a subset of possible S3-compatible systems.
+You may need to consult documentation for your provider to perform equivalent steps.
```

Comment:
We should include a list up here detailing the known-good S3-compatible providers and linking to their documentation.